src/app/routes/stake/stake-list/stake-list.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="stake-list">
+    <h4 class="color-white">Providers</h4>
+
+    <img src="/assets/img/icons/stake-list-banner.svg" alt="stake-list-banner" class="stake-list-banner">
+
+    <div class="row">
+        
+        <div class="col col-md-3" *ngFor="let data of mockData">
+            <app-stake-list-card
+                organization={{data.organization}}
```

Comment:
Like earlier, please use brackets and if you want to be sure that missing data won't throw an error, then add elvis operator. Example `[providerDate]="data?.providerDate`.